src/lib/airtable/request.js

```diff
@@ -592,32 +617,21 @@ export const updateManyTariffPlans = async (recordUpdates) => {
 };
 
 // NONGENERATED: Update the customer record, create a customer update, and update the record in redux
-export const updateCustomer = async (customer, customerUpdate) => {
+export const updateCustomer = async (id, recordUpdates, customerUpdate) => {
   try {
-    const { name, meterNumber, tariffPlanId, siteId, isactive, hasmeter } = customer;
-    const { dateUpdated, customerId, explanation, userId } = customerUpdate;
-    await updateRecord(Tables.Customers, customer.id, {
-      name,
-      meterNumber,
-      tariffPlanId,
-      siteId,
-      isactive,
-      hasmeter,
-    });
+    await updateRecord(Tables.Customers, id, recordUpdates);
     console.log("Customer edited!");
+  } catch (error) {
+    console.log("An error occurred while updating the customer: ", error);
+  }
+  updateCustomerInRedux({ ...recordUpdates, id });
 
-    const updateId = await createCustomerUpdate({
-      dateUpdated,
-      customerId,
-      explanation,
-      userId
-    });
-    console.log("Update id: ", updateId);
-    console.log("Created updates!");
-
-    updateCustomerInRedux(customer);
-  } catch (err) {
-    console.log(err);
+  try {
+    delete customerUpdate.id;
+    await createCustomerUpdate(customerUpdate);
+    console.log("Created Customer update!");
+  } catch (error) {
+    console.log("AN error occurred while creating a customer update: ", error)
```

Comment:
Maybe this is something we can just address all at once later, but it might be good to standardize our error message formatting since we're using pretty different things throughout e.g.

> (Meter Reading) Error occurred when creating meter reading: 

> (createPurchaseRequestAndUpdateInventory) Error: 

> An error occurred while updating the customer: